src/components/Article.jsx

```diff
@@ -95,6 +94,15 @@ console.log(user)
     }
 
 
+    function handleDelete(id) {
+        deleteComment(id).then(() => {
```

Comment:
The logged in user should only be able to delete their own comments. Please implement this.